### PR TITLE
don't fail if unable to find media map for a question

### PR DESCRIPTION
**Pull request by czue**
http://manage.dimagi.com/default.asp?176097

### CODE REVIEW

**Comment by czue** [comment]
the two-space tabs hurt my eyes, but i spent more time fiddling with whitespace than i did implementing the fix before bailing

**Comment by czue** [comment]
@emord buddy

File: touchforms/formplayer/static/formplayer/script/fullform-ui.js [hunk 1]
```diff
@@ -394,22 +394,31 @@ function Question(json, parent) {
...
-          var $widget = self.$container.find('#widget');
-          if ($widget.length) {
-            $widget.append($mediaContainer);
-          } else {
-            $capt.append($mediaContainer);
+    var add_multimedia = function (attrib, control) {
+      var getMediaSrc = function (attrib) {
+        if (self.hasOwnProperty(attrib) && self[attrib]) {
+          try {
+            return getForm(self).adapter.render_context.resourceMap(self[attrib]);
```
Comment (benrudolph) [inline touchforms/formplayer/static/formplayer/script/fullform-ui.js]: this kind of seems like a hammer. if anything were to break in the resourceMap function we'd never know. kind of like using `except Exception`. shouldn't the change be made in the `resourceMap`? https://github.com/dimagi/commcare-hq/blob/fad41eb69d050c6e773db125fe25b85e192b2469/corehq/apps/cloudcare/static/cloudcare/js/backbone/apps.js#L703

**Comment by benrudolph** [comment]
haha that sounds like `fullform-ui.js`!

File: touchforms/formplayer/static/formplayer/script/fullform-ui.js (hunk 1 above)
Comment (benrudolph) [inline touchforms/formplayer/static/formplayer/script/fullform-ui.js]: maybe as a halfway, we could at least log the error in the catch function `console.error(err)`

Comment (czue) [inline touchforms/formplayer/static/formplayer/script/fullform-ui.js]: legit. updated

**Pull request merged**